### PR TITLE
Update numpy.rst

### PR DESCRIPTION
**Pull request by MZuhairKhan**
Fixed solution to Exercise 5, Q 5. 
trun -> trunc
Additionally, added the solutions from when the example code is run.

### CODE REVIEW

**Comment by bast** [comment]
Thanks! Should it use the symbol `#` instead of `//`?

**Comment by MZuhairKhan** [comment]
> Thanks! Should it use the symbol `#` instead of `//`?

I think either works, but I don't know what is the standard on scicomp.

File: content/numpy.rst [hunk 1]
```diff
@@ -465,10 +465,11 @@ Additional exercises
...
        a = np.array([-3.3, -2.5, -1.5, -0.75, -0.5, 0.5, 0.75, 1.5, 2.5, 3])
-       np.round(a)
-       np.ceil(a)
-       np.floor(a)
-       np.trun(a)
+       np.round(a) // [-3. -2. -2. -1. -0.  0.  1.  2.  2.  3.]
+       np.ceil(a) // [-3. -2. -1. -0. -0.  1.  1.  2.  3.  3.]
+       np.floor(a) // [-4. -3. -2. -1. -1.  0.  0.  1.  2.  3.]
+       np.trunc(a) // [-3. -2. -1. -0. -0.  0.  0.  1.  2.  3.]
+
```
Comment (bast) [inline content/numpy.rst:472]:
```suggestion
       np.round(a)  # [-3. -2. -2. -1. -0.  0.  1.  2.  2.  3.]
       np.ceil(a)  # [-3. -2. -1. -0. -0.  1.  1.  2.  3.  3.]
       np.floor(a)  # [-4. -3. -2. -1. -1.  0.  0.  1.  2.  3.]
       np.trunc(a)  # [-3. -2. -1. -0. -0.  0.  0.  1.  2.  3.]

```

Comment (bast) [reply content/numpy.rst:472]: `//` is a division symbol in my understanding but it is very possible that I miss something.

Comment (MZuhairKhan) [reply content/numpy.rst:472]: > `//` is a division symbol in my understanding but it is very possible that I miss something.

Good point, I made this change very quickly in a default text editor without double checking the python. # are the way to go, thanks for pointing it out!

**Comment by bast** [comment]
Thanks again for the fix! I forgot about it and only found it now.

**Pull request merged**